libraries/lib-network-manager/curl/CurlResponse.cpp

```diff
@@ -256,10 +230,9 @@ uint64_t CurlResponse::readData (void* buffer, uint64_t maxBytesCount)
     return maxBytesCount;
 }
 
-void CurlResponse::setPayload(const void* ptr, size_t size)
+void CurlResponse::setPayload(RequestPayloadStreamPtr payload)
 {
-   mPayload = ptr;
-   mPayloadSize = size;
+    mPayload = std::move(payload);
```

Comment:
Indentation, here and in few other places in the `Adds RequestPayload class` commit